.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+version: 2.1
+orbs:
+  codecov: codecov/codecov@1.0.2 # to get this to work, had to opt-in to using third party orbs in Organization Security settings.
+jobs:
+  puget:
+    parameters:
+      python_version:
+        type: string
+    docker:
+      - image: continuumio/miniconda:latest
+    environment:
+      PYTHON: << parameters.python_version >>
+      ENV_NAME: puget_tests
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps-{{ .Branch }}-{{ checksum ".circleci/puget_tests.yml" }}
+      - run:
+          name: configure conda environment
+          command: ./.circleci/install-circle.sh
+      - run:
+          name: install
+          command: |
+            source activate ${ENV_NAME}
+            pip install recordlinkage
+            pip install .
+      - run:
+          name: run puget tests
+          command: |
+            source activate ${ENV_NAME}
+            mkdir test-reports
+            python -m pytest puget -v --cov=puget --cov-config=.coveragerc --cov-report xml:./coverage.xml --junitxml=test-reports/xunit.xml
```

Comment:
I see that codecov uses the coverage.xml output. Who uses the xunit.xml output?

Reply:
I'm not sure, maybe no one. This was something I copied from some other project for another repo and just copied here too. I can take it out and find out what happens.

Reply:
The ol' cargo cult of OSS engineering.

Reply:
I took it out and everything worked, so seems ok.